UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -601,8 +601,9 @@ public void CmdToggleShutters(GameObject switchObj)
 		}
 		else
 		{
-			Logger.LogWarning("player attempted to interact with shutter switch through wall," +
-				" this could indicate a hacked client.");
+			//TODO: Output name of player attempting to interact with shutter switch.
+			Logger.LogWarning("Player attempted to interact with shutter switch through wall," +
```

Comment:
This log message doesn't have a category.

Reply:
> There seem to be multiple log messages where you didn't assign a category.

You're right, silly me. Thanks for letting me know!